src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -177,10 +177,28 @@ const CanBuyResult: FunctionComponent<{
       <CustomerCard nric={nric}>
         <View style={styles.resultWrapper}>
           {Object.entries(cart)
-            .sort()
+            .sort((itemOne, itemTwo) => {
+              const itemOneName = getProduct(itemOne[0])?.name || "";
+              const itemOneOrder = itemOneName.slice(
+                0,
+                itemOneName.indexOf(" ")
+              );
+              const itemTwoName = getProduct(itemTwo[0])?.name || "";
+              const itemTwoOrder = itemTwoName.slice(
+                0,
+                itemTwoName.indexOf(" ")
+              );
+
+              return parseInt(itemOneOrder) - parseInt(itemTwoOrder);
+            })
             .map(([category, canBuy]) => {
               const product = getProduct(category);
-              const categoryText = product?.name || category;
+              let categoryText;
```

Comment:
As we prepend the number, we'll have to remove it before displaying it

Reply:
As mentioned above, we can process whatever is given by the backend so that these components don't need to deal with it

Reply:
ok!

Reply:
Made the changes!

Reply:
👍